### DISCUSSION TITLE
Is there anyway to have a print friendly view for a memo?

### DISCUSSION THREAD

**Discussion by zezretro** [usememos/memos#3937] [Q&A]
Love the app, I am writing each day in it

I need to print some of my notes and the default print view includes the side bars and doesn't format well

Is there some way to create a view suitable for printing?

My solution so far has been paste the markdown into google docs but its a bit tedious

thank you!

**Comment by zezretro** [comment]
One way I have managed to achieve this is using the https://github.com/eallion/memos.top project.

I adapted the CSS to display the notes full page. I then mark the note I want to print as "public" and it shows up.

This helps, My next step would be to write my own PHP script which accepts note ID and formats the page nicely

**Comment by RoccoSmit** [comment]
it looks like you are comfortable changing CSS so I would suggest adding some custom CSS in `settings` -> `system` -> `additional styles`

e.g. 
```
@media print {
  .group.flex.flex-col.justify-start.items-start.fixed.top-0.left-0.select-none, .sticky.top-0.left-0.shrink-0.-mt-6.w-56.h-full {
    display: none;
}
```

The print media query applies the styles only to print view. The above CSS should remove the left and right columns content.
in your browser developer tools (I use Firefox) you should see a toggle to view your page as print view for testing

![image](https://github.com/user-attachments/assets/e05231f8-9a2f-474e-8cf1-d016032ac2c2)

**Reply by zezretro** [reply]
Thank you for your input, I appreciate it. I see where you are going and I implemented your change.

It did have the effect of removing the side bars but it did not let the middle column expand to take up the space

![Screenshot 2024-09-19 at 21 53 13](https://github.com/user-attachments/assets/76e3c63d-3d17-418c-818f-f8791ee0e511)
